src/pjs/pjs.proto

```diff
@@ -137,6 +147,11 @@ service API {
   rpc ListQueue(ListQueueRequest) returns (stream ListQueueResponse) {}
   // InspectQueue returns detailed information about a queue.
   rpc InspectQueue(InspectQueueRequest) returns (InspectQueueResponse) {}
+  // Await blocks until the job has entered or passed the desired state.
+  // Await returns the actual state of the job that met the criteria.
+  // Await can timeout with DEADLINE_EXCEEDED.  In this case clients may
+  // retry in a new request.
+  rpc Await(AwaitRequest) returns (AwaitResponse){}
```

Comment:
Seems like we normally autoformat a space between the response type and `{}`.  clang-format is what you want to run on save here.